packages/frontend/components/composed/journal/CurrentIssue/CurrentIssue.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { graphql } from "react-relay";
+import { useTranslation } from "react-i18next";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import * as Styled from "./CurrentIssue.styles";
+import { CoverImage, ArrowLink, PrecisionDate } from "components/atomic";
+import ContributorsList from "components/composed/contributor/ContributorsList";
+import { CurrentIssueFragment$key } from "@/relay/CurrentIssueFragment.graphql";
+
+export default function CurrentIssue({ data }: Props) {
+  const { t } = useTranslation();
+  const result = useMaybeFragment(fragment, data);
+  const issue = result?.edges[0].node;
+  const articles = issue?.items.edges.slice(0, 3);
+
+  return issue ? (
+    <section className="a-bg-neutral00">
+      <Styled.Inner className="l-container-wide l-grid">
+        <Styled.Left className="l-grid__item l-grid__item--3">
```

Comment:
tiny nit: To keep things consistent, define `Left`'s grid column in the styles.ts file (just like Styled.Right)